tests/integration/upgrade_test.go

```diff
@@ -155,6 +155,9 @@ func (s *UpgradeSuite) updateClusterRoles() error {
 	if _, err := s.k8sh.DeleteResource("ClusterRole", "rook-ceph-global"); err != nil {
 		return err
 	}
+	if _, err := s.k8sh.DeleteResource("ClusterRole", "rook-ceph-cluster-mgmt"); err != nil {
```

Comment:
Do we also need to add the rook-ceph-system role? Looks like there is a change there as well.

Reply:
added